src/client/app/actions/groups.ts

```diff
@@ -271,7 +271,7 @@ function submitNewGroup(group: t.GroupData): Thunk {
 		} catch (e) {
 			dispatch(markGroupInEditingNotSubmitted());
 			if (e.response !== undefined && e.response.status === 400) {
-				showErrorNotification(e.response.data.error);
+				showErrorNotification(translate('400'));
```

Comment:
I'm unclear on why the new, fixed message is better than the old one that gave the specific error that happened. I can see it translates but does it potentially show less information?